src/baseline_rule.py

```diff
@@ -121,22 +138,115 @@ def get_port_array(self):
         if not self.port_min:
             return []
         ports_array = []
-        for port in range(self.port_min, self.port_max + 1):
-            port_rec = {'protocol': self.protocol} if self.protocol else {}
-            port_rec['port'] = port
-            ports_array.append(port_rec)
+        port_rec = {'protocol': self.protocol} if self.protocol else {}
```

Comment:
The code (also the old code) doesn't properly handle the case where `protocol` is set, but `port_min` is not set.
I think the code for Calico works well.

Reply:
done